diff -u -ur FreeFileSync_9.6_Source.orig/FreeFileSync/Source/lib/versioning.cpp FreeFileSync_9.6_Source/FreeFileSync/Source/lib/versioning.cpp
--- FreeFileSync_9.6_Source.orig/FreeFileSync/Source/lib/versioning.cpp	2017-12-07 14:41:15.000000000 +0100
+++ FreeFileSync_9.6_Source/FreeFileSync/Source/lib/versioning.cpp	2017-12-10 12:31:56.892315177 +0100
@@ -1,3 +1,4 @@
+#include <zen/warn_static.h>
 #include "versioning.h"
 #include <cstddef> //required by GCC 4.8.1 to find ptrdiff_t
 
diff -u -ur FreeFileSync_9.6_Source.orig/FreeFileSync/Source/Makefile FreeFileSync_9.6_Source/FreeFileSync/Source/Makefile
--- FreeFileSync_9.6_Source.orig/FreeFileSync/Source/Makefile	2017-12-07 14:41:14.000000000 +0100
+++ FreeFileSync_9.6_Source/FreeFileSync/Source/Makefile	2017-12-10 12:10:43.344878854 +0100
@@ -1,5 +1,5 @@
 APPNAME     = FreeFileSync
-prefix      = /usr
+prefix      = /app
 BINDIR      = $(DESTDIR)$(prefix)/bin
 SHAREDIR    = $(DESTDIR)$(prefix)/share
 APPSHAREDIR = $(SHAREDIR)/$(APPNAME)
@@ -122,5 +122,5 @@
 	$(APPSHAREDIR)
 
 	mkdir -p $(DOCSHAREDIR)
-	cp ../Build/Changelog.txt $(DOCSHAREDIR)/changelog
+	cp ../../Changelog.txt $(DOCSHAREDIR)/changelog
 	gzip $(DOCSHAREDIR)/changelog
diff -u -ur FreeFileSync_9.6_Source.orig/FreeFileSync/Source/RealTimeSync/Makefile FreeFileSync_9.6_Source/FreeFileSync/Source/RealTimeSync/Makefile
--- FreeFileSync_9.6_Source.orig/FreeFileSync/Source/RealTimeSync/Makefile	2017-12-07 14:41:14.000000000 +0100
+++ FreeFileSync_9.6_Source/FreeFileSync/Source/RealTimeSync/Makefile	2017-12-10 12:10:43.347878860 +0100
@@ -1,5 +1,5 @@
 APPNAME = RealTimeSync
-prefix = /usr
+prefix = /app
 BINDIR = $(DESTDIR)$(prefix)/bin
 
 CXXFLAGS  = -std=c++14 -pipe -DWXINTL_NO_GETTEXT_MACRO -I../../.. -I../../../zenXml -include "zen/i18n.h" -Wall -O3 -DNDEBUG `wx-config --cxxflags --debug=no` -pthread
diff -u -ur FreeFileSync_9.6_Source.orig/FreeFileSync/Source/ui/main_dlg.cpp FreeFileSync_9.6_Source/FreeFileSync/Source/ui/main_dlg.cpp
--- FreeFileSync_9.6_Source.orig/FreeFileSync/Source/ui/main_dlg.cpp	2017-12-07 14:41:15.000000000 +0100
+++ FreeFileSync_9.6_Source/FreeFileSync/Source/ui/main_dlg.cpp	2017-12-10 12:15:18.075768416 +0100
@@ -1056,7 +1056,7 @@
         history.resize(globalSettings.gui.cfgFileHistMax);
 
     globalSettings.gui.cfgFileHistory = history;
-    globalSettings.gui.cfgFileHistFirstItemPos = m_listBoxHistory->GetTopItem();
+    // globalSettings.gui.cfgFileHistFirstItemPos = m_listBoxHistory->GetTopItem();
     //--------------------------------------------------------------------------------
     globalSettings.gui.lastUsedConfigFiles.clear();
     for (const Zstring& cfgFilePath : activeConfigFiles_)
@@ -4883,13 +4883,13 @@
 
     m_menuItemCheckVersionAuto->Check(updateCheckActive(globalCfg_.gui.lastUpdateCheck));
 
-    if (shouldRunAutomaticUpdateCheck(globalCfg_.gui.lastUpdateCheck))
+    /* if (shouldRunAutomaticUpdateCheck(globalCfg_.gui.lastUpdateCheck))
     {
         flashStatusInformation(_("Searching for program updates..."));
         //synchronous update check is sufficient here:
         automaticUpdateCheckEval(this, globalCfg_.gui.lastUpdateCheck, globalCfg_.gui.lastOnlineVersion,
                                  automaticUpdateCheckRunAsync(automaticUpdateCheckPrepare().get()).get());
-    }
+    } */
 }
 
 
@@ -4898,7 +4898,7 @@
     //execute just once per startup!
     Disconnect(wxEVT_IDLE, wxIdleEventHandler(MainDialog::OnRegularUpdateCheck), nullptr, this);
 
-    if (shouldRunAutomaticUpdateCheck(globalCfg_.gui.lastUpdateCheck))
+    /* if (shouldRunAutomaticUpdateCheck(globalCfg_.gui.lastUpdateCheck))
     {
         flashStatusInformation(_("Searching for program updates..."));
 
@@ -4910,7 +4910,7 @@
             automaticUpdateCheckEval(this, globalCfg_.gui.lastUpdateCheck, globalCfg_.gui.lastOnlineVersion,
                                      resultAsync.get()); //run on main thread:
         });
-    }
+    } */
 }
 
 
diff -u -ur FreeFileSync_9.6_Source.orig/FreeFileSync/Source/ui/progress_indicator.cpp FreeFileSync_9.6_Source/FreeFileSync/Source/ui/progress_indicator.cpp
--- FreeFileSync_9.6_Source.orig/FreeFileSync/Source/ui/progress_indicator.cpp	2017-12-07 14:41:15.000000000 +0100
+++ FreeFileSync_9.6_Source/FreeFileSync/Source/ui/progress_indicator.cpp	2017-12-10 12:31:11.292277239 +0100
@@ -4,6 +4,7 @@
 // * Copyright (C) Zenju (zenju AT freefilesync DOT org) - All Rights Reserved *
 // *****************************************************************************
 
+#include <zen/warn_static.h>
 #include "progress_indicator.h"
 #include <memory>
 #include <wx/imaglist.h>
diff -u -ur FreeFileSync_9.6_Source.orig/zen/file_io.cpp FreeFileSync_9.6_Source/zen/file_io.cpp
--- FreeFileSync_9.6_Source.orig/zen/file_io.cpp	2017-12-07 14:41:15.000000000 +0100
+++ FreeFileSync_9.6_Source/zen/file_io.cpp	2017-12-10 12:32:59.676404867 +0100
@@ -4,6 +4,7 @@
 // * Copyright (C) Zenju (zenju AT freefilesync DOT org) - All Rights Reserved *
 // *****************************************************************************
 
+#include <zen/warn_static.h>
 #include "file_io.h"
 #include "file_access.h"
 
diff -u -ur FreeFileSync_9.6_Source.orig/zen/shell_execute.h FreeFileSync_9.6_Source/zen/shell_execute.h
--- FreeFileSync_9.6_Source.orig/zen/shell_execute.h	2017-12-07 14:41:15.000000000 +0100
+++ FreeFileSync_9.6_Source/zen/shell_execute.h	2017-12-10 12:10:43.348878862 +0100
@@ -7,6 +7,7 @@
 #ifndef SHELL_EXECUTE_H_23482134578134134
 #define SHELL_EXECUTE_H_23482134578134134
 
+#include "warn_static.h"
 #include "file_error.h"
 
     #include <unistd.h> //fork()
